ui/pages/institutional/pin-mmi-billboard/pin-mmi-billboard.js

```diff
@@ -0,0 +1,597 @@
+import React from 'react';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+export default function OnboardingPinMmiBillboard() {
```

Comment:
It seems like our current pin extension svg exists in images as a static resource.

Reply:
Why do we need to use the context in here?

Reply:
The current MM image lives in: ui/pages/onboarding-flow/pin-extension/pin-billboard.js and it uses strings of text like   `{t('onboardingPinExtensionBillboardTitle')}`.